spec.html

```diff
@@ -41122,7 +41123,7 @@ <h1>%TypedArray%.prototype.slice ( _start_, _end_ )</h1>
               1. Let _srcByteOffset_ be _O_.[[ByteOffset]].
               1. Let _srcByteIndex_ be (_k_ × _elementSize_) + _srcByteOffset_.
               1. Let _targetByteIndex_ be _A_.[[ByteOffset]].
-              1. Let _limit_ be _targetByteIndex_ + min(_count_, _len_) × _elementSize_.
+              1. Let _limit_ be _targetByteIndex_ + _count_ × _elementSize_.
```

Comment:
Always make precedence explicit.

```suggestion
              1. Let _limit_ be _targetByteIndex_ + (_count_ × _elementSize_).
```

Reply:
Done.